.github/workflows/release_charts.yaml

```
@@ -0,0 +1,29 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - rockcraft
+
```

```suggestion
    paths:
      - "deploy/charts/**"
```

This should make sure we do/trigger a release only if something is changed under `deploy/charts`.